### PR TITLE
Ephemeral task update

### PR DESCRIPTION
**Pull request by tom-con**
I can rename the files, I figured I would leave it like this until seed data deployments though.

### CODE REVIEW

File: jakelib/utils.js [hunk 1]
```diff
@@ -67,6 +67,23 @@ const getBranchOrTag = async (service_name) => {
 
 const getUsername = () => GITHUB_USERNAME;
 
+const replacer = (value, variables) => {
```
Comment (siennamw) [inline jakelib/utils.js:70]: I don't see this being used anywhere.  What is it for?

File: jakelib/pipeline.jake [hunk 2]
```diff
@@ -33,36 +34,40 @@ namespace('pipeline', function () {
...
-      throw new Error(`environment and cluster MISMATCH "${cluster}" is not a valid "${environment}" cluster`)
+    if (environment !== 'eph') {
+      try {
+        let a = CLUSTERS[environment][cluster].protected;
+      } catch (e) {
+        throw new Error(`environment and cluster MISMATCH "${cluster}" is not a valid "${environment}" cluster`)
+      }
+  
+      const IS_PROTECTED_TARGET = CLUSTERS[environment][cluster].protected;
+      const IS_RELEASE_BRANCH = toBranch.includes('release/') || toBranch.includes('hotfix/');
```
Comment (siennamw) [inline jakelib/pipeline.jake:45]: didn't realize that this was the actual check for release branches.  seems important to know that a typo in `hotfix/` or `release/` will block a deployment!  😅

**Comment by Brady-Conn** [comment]
👍

File: jakelib/deploy-eph-2.jake [hunk 3]
```diff
@@ -0,0 +1,87 @@
+const { serviceToPath } = require('./utils');
+
+const ECR_URL = '052248958630.dkr.ecr.us-west-2.amazonaws.com';
```
Comment (siennamw) [inline jakelib/deploy-eph-2.jake:3]: is this something that should be hard-coded, or should it go in the .env?

**Pull request merged**